app/assets/javascripts/discourse/mixins/card-contents-base.js.es6

```diff
@@ -31,6 +31,13 @@ export default Ember.Mixin.create({
       return false;
     }
 
+    // Don't show nested
+    if ($target.parents('.card-content').length) {
```

Comment:
This can probably be merged with the previous condition ("Don't show on mobile").